.github/workflows/linux_arm64.yml

```diff
@@ -47,15 +48,40 @@ jobs:
           echo "${{ toJSON(steps.smart_ci.outputs.affected_components) }}"
         shell: bash
 
-  Build:
+  # TODO: Make Docker job a part of Smart CI maybe?
+  Docker:
     needs: Smart_CI
+    runs-on: aks-linux-16-cores-arm-docker-build
+    container:
+      image: openvinogithubactions.azurecr.io/docker_build:0.2
+      volumes:
+        - /mount:/mount
+    outputs:
+      images: "${{ steps.handle_docker.outputs.images }}"
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - uses: ./.github/actions/handle_docker
+        id: handle_docker
+        with:
+          images: |
+            ov_build/ubuntu_20_04_arm64
+            ov_test/ubuntu_20_04_arm64
+          registry: 'openvinogithubactions.azurecr.io'
+          dockerfiles_root_dir: '.github/dockerfiles'
+          changed_components: ${{ needs.Smart_CI.outputs.changed_components }}
+          repo_token: ${{ secrets.GITHUB_TOKEN }}
+
+  Build:
+    needs: [ Smart_CI, Docker ]
     timeout-minutes: 150
     defaults:
       run:
         shell: bash
     runs-on: 'aks-linux-16-cores-arm'
     container:
-      image: openvinogithubactions.azurecr.io/dockerhub/ubuntu:20.04
+      image: ${{ fromJSON(needs.docker.outputs.images).ov_build.ubuntu_20_04_arm64 }}
       volumes:
         - /mount:/mount
       options: -e SCCACHE_AZURE_BLOB_CONTAINER -e SCCACHE_AZURE_CONNECTION_STRING
```

Comment:
let's drop some vars below like `DEBIAN_FRONTEND`, since they are already part of docker file